### ISSUE TITLE
#61 and #68 does not help me

### ISSUE THREAD

**Issue by WhatFreshHellAwaits** [QIDITECH/QIDI_MAX3#70]
I just barely got this printer about a week ago. Now I'm getting this error. All I was doing was following the on screen directions for changing out the filament. I emailed in to support, but found this site after. So trying both for now. This error will not even let me do a factory reset, nor will it pick up the USB in an attempt to reload the firmware. None of the commands on the other post will successfully run either.

QIDI X-MAX3

MCU Protocol error

This is frequently caused by running an older version of the
firmware on the MCU(s). Fix by recompiling and flashing the
firmware.

Your Klipper version is: v0.11.0-6-g336cc92a
MCU(s) which should be updated:
mcu: Current version v0.10.0-530-g3387a9c2-dirty-20221121_013639-mkspi
MKS_THR: Current version v0.10.0-530-g3387a9c2-dirty-20221210_215217-mkspi
Up-to-date MCU(s):

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.

mcu 'MKS_THR': Unable to extract params from: query_thermocouple

**Comment by leckert123** [comment]
The toolhead and mainboard can no longer communicate with one another.

This typically happens when one has pushed the update button prompt in the Fluidd interface. That Fluidd update capability was disabled in more recent firmware versions earlier this year.  QIDI removed the update notification to avoid customers being prompted to update to incompatible firmware.  If you did the update, you'll need QIDI to rescue you.

**Comment by WhatFreshHellAwaits** [comment]
> The toolhead and mainboard can no longer communicate with one another.
> 
> This typically happens when one has pushed the update button prompt in the Fluidd interface. That Fluidd update capability was disabled in more recent firmware versions earlier this year. QIDI removed the update notification to avoid customers being prompted to update to incompatible firmware. If you did the update, you'll need QIDI to rescue you.

Awsome... J/K... I don't think I pushed that but I've done dumber things... I was just changing filament when it happened lol.. Thanks for the reply.

**Comment by leckert123** [comment]
I hope I'm wrong, but it's been far too common a problem. If you received a new machine with older firmware that prevented the Klipper update, then that's on QIDI. If it was purchased used, then you just were not aware of it and you are suffering the same fate many have faced.  

That's why some of us have moved our machines to mainline Klipper.  It's difficult and requires significant technical knowledge.  You'll lose some convenience features that make the machine a bit easier to use, but most of those can be coded in with various macros. Again, not for the faint of heart. It's only for a small niche of hardcore maniacs.

Another chance could be that during the filament change, the system file got corrupted, but I don't think that's the problem. You should still be on Klipper version 0.10xxxxx not 0.11xxxx.  The Max3 shipped with a 2 or 3 year old version of Klipper that was customized by QIDI.  If you're on 0.11xxx like you've stated, that's your problem.

**Comment by WhatFreshHellAwaits** [comment]
Yeah it's a brand new machine. I'm not by printer anymore but pretty sure it says its the .10 version on the screen. Only that error says .11. It's pretty weird.

**Comment by leckert123** [comment]
The email support (AMS is After Market Support) will be able to assist, if they are not working with you already. I usually got answers within 12-24 hrs for my Max3 from Linda at max3support@qd3dprinter.com.  Honestly, they are excellent at trying to help, and you are definitely not the first person with this issue.

https://qidi3d.com/pages/warranty-policy-after-sales-support

Max3support@qd3Dprinter.com
Max3AMS@qd3Dprinter.com

**Comment by WhatFreshHellAwaits** [comment]
They replied with instructions... a little SSH and deleting Klipper, uploading Klipper form GitHub, deleting Moonraker, uploading the replacement from Github, then updating to 4.3.15 and printer no longer appears to be a brick...  very interesting lol... I'm not sure why if I'm using the QIDI slicer only, with the QIDI printer, that it would even prompt for any update, or appear to update automatically to what they are calling a 3rd party Klipper software. Not bad response time at 7 hours... that's quicker then some venders reply in my job. Thanks for sharing the above information.

**Comment by leckert123** [comment]
Very happy it worked out. QIDI's customer support is the best I've witnessed in this industry.  They truly try to help and are concerned about customers.

The "update" prompt I mention is not in QIDI slicer. It appeared at the top right of the Fluidd interface page that you see in your web browser. You get this interface when you type in the IP address for your machine, plus 8080 at the end for the port.  

Again, congratulations that you are back up and running. Enjoy printing!  It's an amazing machine. I just put up another print on mine.

**Comment by WhatFreshHellAwaits** [comment]
> Very happy it worked out. QIDI's customer support is the best I've witnessed in this industry. They truly try to help and are concerned about customers.
> 
> The "update" prompt I mention is not in QIDI slicer. It appeared at the top right of the Fluidd interface page that you see in your web browser. You get this interface when you type in the IP address for your machine, plus 8080 at the end for the port.
> 
> Again, congratulations that you are back up and running. Enjoy printing! It's an amazing machine. I just put up another print on mine.

It was updating to the new 4.3.15 when I wrote that... after update I got a whole new surprise... "The System Starts Abnormally!" Waiting on their reply on this one now... I really don't have the patience for this on a week old device lol...

**Comment by QIDITECH** [comment]
I'm very sorry to let you meet so many problems, I have contacted the after-sales service, they will help you solve the problem as soon as possible.

**Comment by WhatFreshHellAwaits** [comment]
> I'm very sorry to let you meet so many problems, I have contacted the after-sales service, they will help you solve the problem as soon as possible.

I have an email in with them right now. However, there is no ticket number to reference the case.

**Comment by arcangelraider** [comment]
> They replied with instructions... a little SSH and deleting Klipper, uploading Klipper form GitHub, deleting Moonraker, uploading the replacement from Github, then updating to 4.3.15 and printer no longer appears to be a brick... very interesting lol... I'm not sure why if I'm using the QIDI slicer only, with the QIDI printer, that it would even prompt for any update, or appear to update automatically to what they are calling a 3rd party Klipper software. Not bad response time at 7 hours... that's quicker then some venders reply in my job. Thanks for sharing the above information.

Hello. I have the same issue, after preesing the update on Fluidd.  Was Qidi able to resolve the problem on you printer?
